jaq-parse/src/token.rs

```diff
@@ -110,23 +110,27 @@ pub fn token() -> impl Parser<char, Token, Error = Simple<char>> {
     let ctrl = one_of("{}()[]:;,?");
 
     // A parser for identifiers and keywords
-    let ident = text::ident().map(|ident: String| match ident.as_str() {
-        "def" => Token::Def,
-        "if" => Token::If,
-        "then" => Token::Then,
-        "elif" => Token::Elif,
-        "else" => Token::Else,
-        "end" => Token::End,
-        "or" => Token::Or,
-        "and" => Token::And,
-        "as" => Token::As,
-        "reduce" => Token::Reduce,
-        "for" => Token::For,
-        "foreach" => Token::Foreach,
-        "try" => Token::Try,
-        "catch" => Token::Catch,
-        _ => Token::Ident(ident),
-    });
+    let ident = just('@')
+        .or_not()
+        .chain::<char, String, _>(text::ident())
+        .collect()
```

Comment:
To reduce the size of this change, could you terminate the definition of ident after collect(), then redefine ident using the previously defined variable ident?